datetime_test.go

```diff
@@ -258,24 +258,14 @@ func TestDateTime_InRange(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			reporter := newMockReporter(t)
 
-			if tc.expectInRange {
-				NewDateTime(reporter, tc.value).
-					InRange(tc.min, tc.max).
-					chain.assertNotFailed(t)
-			} else {
-				NewDateTime(reporter, tc.value).
-					InRange(tc.min, tc.max).
-					chain.assertFailed(t)
-			}
-			if tc.expectNotInRange {
-				NewDateTime(reporter, tc.value).
-					NotInRange(tc.min, tc.max).
-					chain.assertNotFailed(t)
-			} else {
-				NewDateTime(reporter, tc.value).
-					NotInRange(tc.min, tc.max).
-					chain.assertFailed(t)
-			}
+			NewDateTime(reporter, tc.value).
+				InRange(tc.min, tc.max).
+				chain.assertOK(t, tc.expectInRange)
```

Comment:
nit: could you please reformat everything to a more compact 2-line format like this:

```go
			NewDateTime(reporter, tc.value).InRange(tc.min, tc.max).
				chain.assertOK(t, tc.expectInRange)
```